HISTORY.md

```diff
@@ -8,9 +8,13 @@ unreleased
   * deps: debug@2.6.1
     - Allow colors in workers
     - Deprecated `DEBUG_FD` environment variable set to `3` or higher
+    - Fix DEBUG_MAX_ARRAY_LENGTH
+    - Fix Electron reference to `process.env.DEBUG`
     - Fix error when running under React Native
+    - Fix Inline extend function in node implementation
+    - Fix null reference check on window.documentElement.style.WebkitAppearance
     - Use same color for same namespace
-    - deps: ms@0.7.2
+    - deps: ms@2.0.0 to fix regular expression denial of service vulnerability
```

Comment:
We don't list changes in sub dependencies, and besides, since this only explains what our user's will see in the upgrade, they don't see a vulun fix since the vulun code was not being used. Just change to `deps: ms@2.0.0`.